microsoft-365/admin/create-groups/office-365-groups-expiration-policy.md

```diff
@@ -73,7 +76,20 @@ Group owners will only be notified via email. If the group was created via Plann
 
 30 days before the group expires, the group owners (or the email addresses that you specified for groups that don't have an owner) will receive an email allowing them to easily renew the group. If they don't renew it, they'll receive another renewal email 15 days before expiration. If they still haven't renewed it, they will receive one more email notification the day before expiration.
 
-If for some reason none of the owners or admins renew the group before it expires, the admin can still restore the group for up to 30 days after expiration. For details see: [Restore a deleted Microsoft 365 group](https://docs.microsoft.com/microsoft-365/admin/create-groups/restore-deleted-group).
+If for some reason none of the owners or admins renew the group before it expires, and automatic renewal does not occur due to the group not meeting the requirements to be automatically renewed, the admin can still restore the group for up to 30 days after expiration. For details see: [Restore a deleted Microsoft 365 group](https://docs.microsoft.com/microsoft-365/admin/create-groups/restore-deleted-group).
+
+## How automatic renewal fits into the group expiration timeline
+
+Automatic renewal checks occur starting 40 days in advance from the group's expiration date. The service will continually check for activity from this point until the group expires, is manually renewed, or meets the requirements for automatic renewal and is thus renewed. If the service determines the group has been active at any point during the group's lifecycle, the service will begin the automatic renewal process. For example, if activity occurred 60 days from expiration, the activity check performed 40 days from expiration would indicate the group is active, and the automatic group renewal process would immediately begin.
```

Comment:
Can we rephrase this as follows:
Automatic renewal checks occur continuously during the Group's lifecycle. If there is any activity in the group prior to 30 days of expiry then the group will be auto-renewed and group owners will not receive any e-mail notifications for manual group renewal.